content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -74,30 +90,38 @@ In certain cases, there is an exception to the status calculation for monitor-ba
 - Wait until a specified number of the groups are failing (default: 1)
 - Retry a specified number of times before a location's test is considered a failure (default: 0)
 
-By changing any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using just that one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups. 
+If you change any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups.
+
+For more information on Synthetic test alerting conditions, see the Synthetic Monitoring [documentation][4].
 
-For more information on Synthetic test alerting conditions, visit the Synthetic Monitoring [documentation][7].
+### Impact of manual and automatic monitor updates
+
+When a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting, SLO calculations do not change. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification` settings, and using the clone for your SLO.
+
+Datadog recommends against using monitors with `Alert Recovery Threshold` and `Warning Recovery Threshold` as they can also affect your SLO calculations and do not allow you to cleanly differentiate between a SLI's good behavior and bad behavior.
+
+Muting a monitor has no effect on the SLO calculation.
+
+To exclude time periods from an SLO calculation, use the [SLO status corrections][5] feature.
```

Comment:
We should add a new subsection here to talk about the impact of the NO DATA state. Ultimately what happens is: if the underlying monitor has notify_no_data set to false then NO DATA is treated as OK by the SLO, if the underlying monitor has notify_no_data set to true then NO DATA is treated as ALERT by the SLO.